src/library/kotlin/org/jetbrains/kotlin/konan/library/SearchPathResolver.kt

```diff
@@ -194,7 +200,10 @@ internal class KonanLibraryProperResolver(
                 .map { it.takeIf { libraryMatch(it, unresolved) } }
                 .filterNotNull()
 
-        return matching.firstOrNull() ?: error("Could not find \"$givenPath\" in ${searchRoots.map { it.absolutePath }}.")
+        return matching.firstOrNull() ?: run {
+            logger.error("Could not find \"$givenPath\" in ${searchRoots.map { it.absolutePath }}.")
+            exitProcess(1)
```

Comment:
Better add `fatal` method returning `Nothing` to `Logger` instead with `exitProcess(1)` as implementation in `DummyLogger`.